reusable/images_to_numpy.py

```diff
@@ -0,0 +1,133 @@
+import os
+import cv2
+import argparse
+import numpy as np
+from tqdm import tqdm
+from PIL import Image
+import matplotlib.pyplot as plt
+
+"""
+Script to convert a folder of images into numpy format.
+
+Folder structure for images:
+Dataset:
+- Images
+    - image_01
+    - image_02
+    - image_03
+
+Usage: 
+python images_to_numpy.py --root_data_path "./Dataset" --image_path "Images" --saved_file_name "to_numpy" --visualize yes --visualize_for multiple 
+"""
+
+parser = argparse.ArgumentParser(description= 'Convert images to numpy array')
+parser.add_argument('--root_data_path', type= str, required= True, default = "./Dataset", help = 'Define root dataset path.')
+parser.add_argument('--image_path', type= str, required= True, default= "Images", help = 'Set images path.')
+parser.add_argument('--saved_file_name', type= str, required= True, default= "images_to_nparray", help = 'Define saved file name.')
+parser.add_argument('--visualize', type= str, choices= ['no', 'yes'], default= "no", required= True, help = 'Visualize images.')
+parser.add_argument('--visualize_for', type= str, choices= ['single', 'multiple', "no"], default="no", required= False, help = 'Choose Visualize for single/multiple images.')
+
+args = parser.parse_args()
+
+
+def crop_and_resize(img, resize_dim = 256):
+    """
+    Resize the input image.
+    
+    Aurguments:
+        img: Orginal input image
+        resized_dim: Size of new image (Default: 256)
+    
+    Returns:
+        img: new image
+    """
+    img = cv2.resize(img, (resize_dim, resize_dim), interpolation=cv2.INTER_AREA)
+    return img
+
+def get_data(images):
```

Comment:
should be image_path and not images, we cannot read images from images hahah

Reply:
Done.